ra/ra.go

```diff
@@ -906,16 +903,10 @@ func (ra *RegistrationAuthorityImpl) issueCertificate(
 	logEvent.CommonName = csr.Subject.CommonName
 	logEvent.Names = csr.DNSNames
 
-	// Validate that authorization key is authorized for all domains
+	// Validate that authorization key is authorized for all domains in the CSR
 	names := make([]string, len(csr.DNSNames))
 	copy(names, csr.DNSNames)
 
-	if len(names) == 0 {
```

Comment:
Why is this check removed?

Reply:
Per [@roland's feedback](https://github.com/letsencrypt/boulder/pull/3252#discussion_r154435153) - it's already done in the CSR verify.